lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyDiagnosticParticipant.java

```diff
@@ -81,7 +81,7 @@ private void validateFeature(DOMDocument domDocument, List<Diagnostic> list, DOM
             DOMNode featureTextNode = (DOMNode) featureNode.getChildNodes().item(0);
             // skip nodes that do not have any text value (ie. comments)
             if (featureTextNode != null && featureTextNode.getTextContent() != null) {
-                String featureName = featureTextNode.getTextContent();
+                String featureName = featureTextNode.getTextContent().trim();
```

Comment:
Is there a test case that can be updated to show this works?

Reply:
Yeah, I can modify one.